docs/usage.md

```diff
@@ -169,8 +169,15 @@ fund performed in epoch 1030279
 ```
 
 The epoch were the message is performed can give you a sense of the time the message will take to be propagated. You can check the current finality in a subnet and wait for the finality height that includes your message to be committed.
+```bash
+./bin/ipc-cli cross-msg parent-finality --subnet <SUBNET_ID>
+```
 
-> TODO: Add a cli command to check the finality committed in a child.
+```console
+# Example execution
+$ ./bin/ipc-cli cross-msg parent-finality --subnet /r31415926/t4xwzbdu7z5sam6hc57xxwkctciuaz7oe5omipwbq --to=0x406a7a1d002b71ece175cc7e067620ae5b58e9ec 100
```

Comment:
Is that a real address? I thought it would be t410 prefix :thinking:

Reply:
Copy-past typo. The `to` shouldn't be a flag there, already fixed. Thanks.